normalize/coll.py

```diff
@@ -460,6 +460,8 @@ def __init__(self, typekey):
         self.typekey = typekey
 
     def __call__(self, values):
+        if six.PY3 and self.typekey[1] == '__builtin__.str':
```

Comment:
do we have any way of knowing if there are any other types starting with __builtin__?